app/src/main/java/org/p2p/wallet/moonpay/ui/new/NewBuyPresenter.kt

```diff
@@ -103,7 +104,9 @@ class NewBuyPresenter(
             view?.showLoading(isLoading = true)
             currentAlphaCode = paymentMethodsInteractor.getBankTransferAlphaCode()
 
-            val availablePaymentMethods = paymentMethodsInteractor.getAvailablePaymentMethods(currentAlphaCode)
+            val availablePaymentMethods = paymentMethodsInteractor.getAvailablePaymentMethods(
+                currentAlphaCode, preselectedMethodType
```

Comment:
to make it clear, let `alphaCode` variables always represent the standard of their value:
```suggestion
                currentAlpha3Code, preselectedMethodType
```